examples/01_standalone_sdk/09_pause_example.py

```diff
@@ -42,27 +41,54 @@
 conversation = Conversation(agent, workspace=os.getcwd())
 
 
-print("Simple pause example - Press Ctrl+C to pause")
+print("=" * 60)
+print("Pause and Continue Example")
+print("=" * 60)
+print()
 
-# Send a message to get the conversation started
-conversation.send_message("repeatedly say hello world and don't stop")
+# Phase 1: Start a long-running task
+print("Phase 1: Starting agent with a task...")
+conversation.send_message(
+    "Create a file called countdown.txt and write numbers from 100 down to 1, "
+    "one number per line. After you finish, summarize what you did."
+)
```

Comment:
Will this actually take 5 seconds? I feel like it might just run a single bash command and then be done.